package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Cell Tower Coverage",
+  "version": "0.1.1",
+  "description": "A starting point for new projects and atlas.utah.gov",
+  "homepage": "https://atlas.utah.gov",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/agrc/AGRCJavaScriptProjectBoilerPlate.git"
+  },
```

Comment:
We've also started to start projects at version 1.0.0. You can read about some of the weirdness around starting with 0 [on this blog](https://nodesource.com/blog/semver-tilde-and-caret/#recommendationstartat100)